SoCCo/algorithms/climate.py

```diff
@@ -1,10 +1,10 @@
-### This folder contains the climate and social functions for the coupled social
-### and climate model
+### This folder contains the climate functions for the coupled social
```

Comment:
Ok, I addressed these two comments above and pushed the modifications on this same branch.